project/LibDependencies.scala

```diff
@@ -1,31 +1,31 @@
 import sbt._
 import PlayCrossCompilation.dependencies
+import play.core.PlayVersion
 import play.sbt.PlayImport.ws
 import sbt.{ModuleID, Test}
 
 object LibDependencies {
 
-  val playFrontendGovukVersion = "0.65.0"
-  val playFrontendHmrcVersion  = "0.52.0"
+  val playFrontendGovukVersion = "0.72.0"
+  val playFrontendHmrcVersion  = "0.65.0"
   val govukFrontendVersion     = "3.11.0"
 
   lazy val libDependencies: Seq[ModuleID] = dependencies(
     shared = {
-      import PlayCrossCompilation.playRevision
-
       val compile = Seq(
-        "com.typesafe.play" %% "play"            % playRevision,
-        "com.typesafe.play" %% "filters-helpers" % playRevision,
+        "com.typesafe.play" %% "play"            % PlayVersion.current,
+        "com.typesafe.play" %% "filters-helpers" % PlayVersion.current,
         "org.webjars.npm"    % "govuk-frontend"  % govukFrontendVersion,
         "org.joda"           % "joda-convert"    % "2.0.2",
         "com.lihaoyi"       %% "fastparse"       % "2.1.2"
       )
 
       val test = Seq(
+        "com.vladsch.flexmark"          % "flexmark-all"     % "0.35.10",
         "org.scalatest"                %% "scalatest"        % "3.0.8",
         "org.pegdown"                   % "pegdown"          % "1.6.0",
```

Comment:
I could be wrong, but might be worth checking if `pegdown` can be removed.

Reply:
Removed for 2.8, but still needed for earlier versions.